drivers/gles3/storage/material_storage.cpp

```diff
@@ -1272,6 +1273,7 @@ MaterialStorage::MaterialStorage() {
 		actions.renames["POINT_COORD"] = "gl_PointCoord";
 		actions.renames["INSTANCE_CUSTOM"] = "instance_custom";
 		actions.renames["SCREEN_UV"] = "screen_uv";
+		actions.renames["REGION_RECT"] = "region_rect";
```

Comment:
Sorry, missed this before. This can be removed since it is for spatial shaders. Only the definition above is needed